CoffeeChat/Utilities/Time+Helper.swift

```diff
@@ -8,15 +8,39 @@
 
 import Foundation
 
+enum Weekday: String {
+    case monday, tuesday, wednesday, thursday, friday, saturday, sunday
+
+    static func fromString(_ day: String) -> Weekday {
+        let day = day.lowercased()
+        if let weekday = Weekday(rawValue: day) {
+            return weekday
+        } else {
+            print("Tried to create a Weekday with \(day); returning .monday instead")
+            return .monday
+        }
+    }
+}
+
 class Time {
 
+    /// Day of the week matches are first assigned
+    static let matchDay: Weekday = .sunday
+
     static let amTimes = ["9:00", "9:30", "10:00", "10:30", "11:00", "11:30"]
     static let pmTimes = [
         "12:00", "12:30", "1:00", "1:30", "2:00", "2:30",
         "3:00", "3:30", "4:00", "4:30", "5:00", "5:30",
         "6:00", "6:30", "7:00", "7:30", "8:00", "8:30"
     ]
 
```

Comment:
Not sure if this is relevant, but are we accounting for different time zones right now?

Reply:
For the MVP, we are only supporting Ithaca time zone